sonar-pmd-lib/src/main/java/org/sonar/plugins/pmd/rule/util/PmdSeverityMapper.java

```diff
@@ -11,29 +13,67 @@ private PmdSeverityMapper() {
 
     /**
      * Converts PMD priority ("1".."5") into Sonar severity string.
-     * Defaults to "MAJOR" when the input is null/empty/unknown.
-     *
-     * PMD priorities:
-     * 1 -> BLOCKER
-     * 2 -> CRITICAL
-     * 3 -> MAJOR
-     * 4 -> MINOR
-     * 5 -> INFO
      *
-     * @param priority PMD priority as string
-     * @return Sonar severity label
+     * @param pmdPriority PMD priority as string
+     * @param category    Category of the PMD rule
+     * @return Sonar severity as defined by {@code org.sonar.api.rule.Severity} (BLOCKER, CRITICAL, MAJOR, MINOR, INFO).
+     * Defaults to "MAJOR" when the input is null/empty/unknown.
+     * <p>
+     * Default Priority Mapping:
+     * For standard PMD rules, priorities are mapped as follows:
+     * <ul>
+     *   <li>1 &rarr; BLOCKER</li>
+     *   <li>2 &rarr; CRITICAL</li>
+     *   <li>3 &rarr; MAJOR</li>
+     *   <li>4 &rarr; MINOR</li>
+     *   <li>5 &rarr; MINOR</li>
+     * </ul>
```

Comment:
For clarity add comment to each of the following: // aka HIGH, // aka MEDIUM, // aka LOW, aka //LOW

Reply:
fixed